test/funnels/Swapper.t.sol

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.16;
+
+import "dss-test/DssTest.sol";
+import { Swapper } from "src/funnels/Swapper.sol";
+import { UniV3SwapperCallee } from "src/funnels/callees/UniV3SwapperCallee.sol";
+import { AllocatorRoles } from "src/AllocatorRoles.sol";
+import { AllocatorBuffer } from "src/AllocatorBuffer.sol";
+import { TestUtils } from "test/utils/TestUtils.sol";
+
+interface GemLike {
+    function balanceOf(address) external view returns (uint256);
+    function transfer(address, uint256) external;
+}
+
+contract CalleeMock is DssTest {
+    function swap(address src, address dst, uint256 amt, uint256, address to, bytes calldata) external {
+        GemLike(src).transfer(address(0xDEAD), amt);
+        deal(dst, address(this), amt, true);
+        GemLike(dst).transfer(to, amt);
+    }
+}
+
+contract SwapperTest is DssTest, TestUtils {
+    event Swap (address indexed sender, address indexed src, address indexed dst, uint256 amt, uint256 out);
```

Comment:
```suggestion
    event Swap(address indexed sender, address indexed src, address indexed dst, uint256 amt, uint256 out);
```

Reply:
Done in https://github.com/makerdao/dss-allocator/pull/15